api/rs/slint/Cargo.toml

```diff
@@ -181,7 +181,7 @@ backend-android-activity-05 = [
 ## Access to these APIS is valuable, but as WGPU releases new major versions frequently, this feature as well as the
 ## APIs guarded with it are *NOT* subject to the usual Slint API stability guarantees. This will may be removed in future
 ## minor releases of Slint, likely to be replaced by a feature with a similar name but the WGPU version suffix being bumped.
```

Comment:
maybe we should then recommand to use slint using the tilde requirement https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#tilde-requirements `version = "~1.12"`

Reply:
Good idea